clients/cli/src/analytics.rs

```diff
@@ -245,8 +241,8 @@ pub async fn track_proof_accepted(
 
 /// Track analytics for proof submission success (non-blocking)
 pub async fn track_proof_submission_success(
```

Comment:
Would it be more engineering-friendly to move `track_proof_submission_success` and `track_authenticated_proof_analytics` to `analytics.rs`?